.travis.yml

```diff
@@ -51,6 +55,7 @@ matrix:
 
 git:
   depth: 1
+  submodules: false # Don't do recursive init
```

Comment:
is this to avoid pulling in all of `libswiftnav`'s deps?

Reply:
Yes, travis default submodule init is recursive. It lead to problems with SSH keys as the `libswiftnav`s submodules are using SSH instead of HTTPS. I didn't dig into details what would have been needed for the SSH approach.